### ISSUE TITLE
Issues with script only stacks and Levure

### ISSUE THREAD

**Issue by EzekTec-Inc** [trevordevore/levure#64]
When I open a Sublime Text project, load Levure (standalone.livecode) and launch my stack (have "go stack "e-Fare" in _**OpenApplication**_ handler), after saving my stack in livecode I find my script only stacks assigned as a background to my home card stops working.

I have troubleshot this issue and can't seem to figure what the issues is, except for something breaking the connections somewhere. When I resort to using script only stacks in Livecode, without levure, I do not get this issue.

I maybe doing something wrong I suppose?

**Comment by trevordevore** [comment]
@EzekTec-Inc A couple of questions:

1. Are you seeing any errors in LiveCode when you save the script in Sublime Text? 
2. If you edit the stack script in the IDE after saving using Sublime Text do you see the changes in the LiveCode script editor? That will help us determine if the changes are actually being made.

**Comment by EzekTec-Inc** [comment]
1. There are no errors firing in Sublime Text when I save the script only stack
2. I do see the changes reflected in LiveCode script editor when I save changes, but notice my changes doesn't get updated (in LiveCode script editor), which prompts me to check if my card base script only stacks is still working (which it's not).

Now, when I close out everything and restart (Sublime and LiveCode), everything seems to work perfectly. After a while, everything returns to the issue state reported earlier.

I have changed ports in Sublime and this doesn't help either (gone as far as isolating a port track it status).

I just found out that, when this happens (card losing connection to its script only stack behavior) and I manually issue a set card behavior in the livecode message console, everything seems to return to normal until I commit changes in Sublime Text and then it all goes out to lunch. This issue does not happen when I use script only stack in LiveCode directly (without levure).
